lib/registries/addon/drafts/draft/route.ts

```diff
@@ -39,9 +43,14 @@ export default class DraftRegistrationRoute extends Route {
 
     model(params: { id: string }): DraftRouteModel {
         const { id: draftId } = params;
+        const taskInstance = this.loadModelTask.perform(draftId);
+        const draftRegistrationManager = new DraftRegistrationManager(taskInstance);
```

Comment:
👀 Something is causing the parent route model to recompute every transition between child routes, which creates new `DraftRegistrationManager` and `NavigationManager` instances.

Reply:
Fix: [turn off refreshModel on `view_only` queryParam](https://github.com/CenterForOpenScience/ember-osf-web/pull/906/files#diff-ec6e5b2f253723290cba636e157707e2R31)